packages/self_storing_input/lib/src/self_storing_text.dart

```diff
@@ -45,16 +45,16 @@ class _SelfStoringTextState extends State<SelfStoringText> {
     super.dispose();
   }
 
-  void _emptySetState() {}
+  void _emptySetState() => setState(() {});
```

Comment:
This is typically a code smell. What state are you trying to set here?

Reply:
_emptySetState is a listener for the object _state, that is shared between widgets and notifies about important changes. 
Any anti-pattern here? What would be alternative?

(I am submitting this, because I want the bug to be fixed. I will apply follow up, if needed, in other PR. )

Reply:
Yes, this is an anti=pattern but no need to panic. The reason why this is an anti-pattern is that it is not obvious what state you are setting. Simply rename this method `_onSharedStateChange` :-)